src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -138,56 +138,15 @@ modalities include:
-key-value pair: `_run-1`, `_run-2`, `_run-3` etc. (only integers are allowed as
-run labels). When there is only one scan of a given type the run key MAY be
-omitted. Please note that diffusion imaging data is stored elsewhere (see
-below).
-
-#### The `acq` entity
-
-The OPTIONAL `acq-<label>` key/value pair corresponds to a custom label the user
-MAY use to distinguish a different set of parameters used for acquiring the same
-modality. For example this should be used when a study includes two T1w images -
-one full brain low resolution and and one restricted field of view but high
-resolution. In such case two files could have the following names:
-`sub-01_acq-highres_T1w.nii.gz` and `sub-01_acq-lowres_T1w.nii.gz`, however the
-user is free to choose any other label than `highres` and `lowres` as long as
-they are consistent across subjects and sessions. In case different sequences
-are used to record the same modality (e.g. RARE and FLASH for T1w) this field
-can also be used to make that distinction. At what level of detail to make the
-distinction (e.g. just between RARE and FLASH, or between RARE, FLASH, and
-FLASHsubsampled) remains at the discretion of the researcher.
-
-#### The `ce` entity
-
-Similarly the OPTIONAL `ce-<label>` key/value can be used to distinguish
-sequences using different contrast enhanced images. The label is the name of the
-contrast agent. The key `ContrastBolusIngredient` MAY be also be added in the
-JSON file, with the same label.
-
-Some meta information about the acquisition MAY be provided in an additional
-JSON file. See [Common metadata fields](#common-metadata-fields) for a
-list of terms and their definitions. There are also some OPTIONAL JSON
-fields specific to anatomical scans:
-
-| Field name              | Definition                                                                                                                                         |
-| -----------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------- |
-| ContrastBolusIngredient | OPTIONAL. Active ingredient of agent. Values MUST be one of: IODINE, GADOLINIUM, CARBON DIOXIDE, BARIUM, XENON Corresponds to DICOM Tag 0018,1048. |
-
-#### The `rec` entity
-
-Similarly, the OPTIONAL `rec-<label>` key/value can be used to distinguish
-different reconstruction algorithms (for example ones using motion correction).
+Currently supported entities include:
+
+-   [`sub-<label>`](../99-appendices/09-entities.md#sub)
+-   [`ses-<label>`](../99-appendices/09-entities.md#ses)
+-   [`acq-<label>`](../99-appendices/09-entities.md#acq)
+-   [`ce-<label>`](../99-appendices/09-entities.md#ce)
+-   [`rec-<label>`](../99-appendices/09-entities.md#rec)
+-   [`run-<index>`](../99-appendices/09-entities.md#run)
+-   [`mod-<label>`](../99-appendices/09-entities.md#mod)
```

Comment:
Does anyone have any thoughts on this approach to linking entity definitions in the datatype sections?

Reply:
It looks quite clean and prevents duplication of information, which is nice.

Yet it also disturbs the reading flow slightly.

Furthermore, when reading a part of the spec, for example on MRI, a given entity can be described with respect to that modality and tune the examples towards that use case. If we describe the entities exclusively in the centralized location, these descriptions would have to have either only very general descriptions/examples, or be very long to make sure all use cases are covered.

Reply:
I definitely want the entity _definitions_ to be general across their applications, but I do see why section-specific examples are important. Part of the problem is that we want as much of the specification to be auto-generated from the schema as possible. This means limiting section-specific text as much as possible. On the other hand, we still need that textual glue to connect all of the auto-generated text within sections, and I'm not sure what the best way is to manage that.

Reply:
Okay, I dropped the awful lists and took a much more limited approach, so I'll resolve this thread.